src/index.ts

```diff
@@ -142,7 +139,10 @@ async function run() {
     );
 
     finalExecutionResult.executions.forEach((execution: Execution) => {
-      prettyPrintExecution(execution);
+      core.setOutput(
```

Comment:
I updated this to with out via `core` compared to `console.log`, since they appear to prefer this.